build.gradle

```diff
@@ -105,6 +105,10 @@ test {
 
 spotless {
 	java {
+		// Specify java source file tree to avoid scanning openapi generated code.
```

Comment:
I hadn't seen this problem before but it happened once, and adding this fixed it.

Reply:
seems like a good idea